lib/grpc/getGrpcError.ts

```diff
@@ -75,9 +75,9 @@ const getGrpcError = (err: any) => {
 
   // return a grpc error with the code if we've assigned one, otherwise pass along the caught error as UNKNOWN
   const grpcError: grpc.ServiceError = {
-    code: code || status.UNKNOWN,
-    message: err.message,
-    name: err.name,
+    code: code ?? status.UNKNOWN,
```

Comment:
The `??` operator is new in TypeScript and I am not too familiar with it yet. What's the difference between those two statements?
- `code: code ?? status.UNKNOWN,`
- `code: code || status.UNKNOWN,`

Reply:
`??` only uses the right side of the expression if the left side is actually `null` or `undefined` and not just falsey. So 

```typescript
const code = 0;
console.log(code || 'unknown'); // "unknown"
console.log(code ?? 'unknown'); // 0
```

I'm aiming to use any of those if they have a value, even if it's a 0 (which happens to be grpc.status `OK`) or an empty string, but if they're missing altogether then I set those defaults.